### PR TITLE
Milestone: Email validation

### PR DESCRIPTION
**Pull request by Roseokpe**
In this milestone of the portfolio website, I implement a client-side validation of the data in the contact form before submission.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @Roseokpe,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors check not passed 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -21,4 +22,21 @@ function hideMenu() {
...
+
+// the function below check if the email address is all lowercase or not //
+function emailCheckLowerCase(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    error.classList.remove('hide');
+    e.preventDefault();
+  } else {
+    error.classList.add('hide');
+  }
+}
```
Comment (m05e5) [inline index.js]: - [ ] Hi great job. Please as mentioned in the requirements the lowercase validation is supposed to check, the email address when the user tries to submit the form.

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Roseokpe , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] _**Please check the comments below**_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
# Approved

Hi @Roseokpe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct Gitflow :heavy_check_mark: 

## Optional suggestions
- [ ] You could consider adding a check for an empty email field upon hitting the submit button

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**